package.json

```diff
@@ -46,19 +50,31 @@
   },
   "jest": {
     "moduleNameMapper": {
-      "@/(.*)$": "<rootDir>/src/$1"
+      "@/(.*)$": "<rootDir>/src/$1",
+      "^jose": "jose"
     },
-    "testEnvironment": "node",
+    "moduleFileExtensions": [
+      "js",
+      "jsx",
+      "tsx",
+      "ts"
+    ],
+    "testEnvironment": "jsdom",
+    "setupFiles": [
+      "./jest.setup.ts"
+    ],
     "transform": {
-      "^.+.tsx?$": [
-        "ts-jest",
-        {}
-      ]
+      "^.+\\.(ts|tsx)?$": "ts-jest"
     },
     "modulePaths": [
       "<rootDir>/src",
       "<rootDir>/node_modules"
-    ]
+    ],
+    "globals": {
+      "ts-jest": {
+        "tsConfig": "tsconfig.jest.json"
+      }
```

Comment:
I get a few warnings when I run jest, just for capitalisation and stuff, can we fix it to silence them